vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -177,33 +172,36 @@ func (d *VolumeDriver) MountVolume(name string, fstype string, id string, isRead
 		return mountpoint, err
 	}
 
-	watcher, skipInotify := fs.DevAttachWaitPrep(name, watchPath)
-
-	// Have ESX attach the disk
-	dev, err := d.ops.Attach(name, nil)
-	if err != nil {
-		return mountpoint, err
+	watchCtx, errWatch := fs.DevAttachWaitPrep()
```

Comment:
It's not handled right away in this scenario. It's [later used](https://github.com/vmware/docker-volume-vsphere/pull/1504/files/9afadf65fc530299df76caad9315b7f4d0e9bfda#diff-aeea88b6324054966e71ce0340771803R195) to check if a `time.Sleep(..)` is needed.